Traceback (most recent call last):
  File "/home/orestas41/project-FootballPredict/lithuanian-football-league-predictor/training_validation/run.py", line 198, in <module>
    go(args)
  File "/home/orestas41/project-FootballPredict/lithuanian-football-league-predictor/training_validation/run.py", line 73, in go
    sk_pipe.fit(X_train, y_train)
  File "/home/orestas41/miniconda3/envs/mlflow-459c2e262f0b90dfe630577ea32ef02f00ebb4fb/lib/python3.10/site-packages/sklearn/pipeline.py", line 394, in fit
    self._final_estimator.fit(Xt, y, **fit_params_last_step)
  File "/home/orestas41/miniconda3/envs/mlflow-459c2e262f0b90dfe630577ea32ef02f00ebb4fb/lib/python3.10/site-packages/sklearn/ensemble/_forest.py", line 327, in fit
    X, y = self._validate_data(
  File "/home/orestas41/miniconda3/envs/mlflow-459c2e262f0b90dfe630577ea32ef02f00ebb4fb/lib/python3.10/site-packages/sklearn/base.py", line 581, in _validate_data
    X, y = check_X_y(X, y, **check_params)
  File "/home/orestas41/miniconda3/envs/mlflow-459c2e262f0b90dfe630577ea32ef02f00ebb4fb/lib/python3.10/site-packages/sklearn/utils/validation.py", line 964, in check_X_y
    X = check_array(
  File "/home/orestas41/miniconda3/envs/mlflow-459c2e262f0b90dfe630577ea32ef02f00ebb4fb/lib/python3.10/site-packages/sklearn/utils/validation.py", line 746, in check_array
    array = np.asarray(array, order=order, dtype=dtype)
  File "/home/orestas41/miniconda3/envs/mlflow-459c2e262f0b90dfe630577ea32ef02f00ebb4fb/lib/python3.10/site-packages/pandas/core/generic.py", line 2072, in __array__
    return np.asarray(self._values, dtype=dtype)
ValueError: could not convert string to float: '2022-04-09 13:00:00'